scripts/src/main/resources/scripts/command/az

```diff
@@ -10,88 +10,45 @@ then
   exit
 fi
 
-if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
 # shellcheck source=scripts/functions
 source "$(dirname "${0}")"/../functions
 
 function doSetup() {
-  if command -v az &> /dev/null
+  if [ "${1}" != "silent" ] || [[ ! $(command -v az) =~ ${DEVON_IDE_HOME}/software/ ]]
   then
-    if [ "${1}" != "silent" ] && ! doIsQuiet
-    then
-      doEcho "Azure CLI is already installed at $(command -v az)"
-    fi
-  else
-    doRequireNotBatch
-    if doIsWindows
-    then
-      # Get leatest release
-      if [ -z "${AZ_VERSION}" ]
-      then
-        doEcho "Getting latest release..."
-        AZ_VERSION=$(curl -s https://api.github.com/repos/Azure/azure-cli/releases/latest | awk -F ":" '/tag_name/ {print $2}'| awk -F "\"" '{print $2}' | awk -F "-" '{print $3}')
-      fi
-
-      doInstall "-" "${AZ_HOME}" "az" "${AZ_VERSION}"
-      if [ "${?}" = 0 ]
-      then
-        doPackageInstall "${AZ_HOME}"
-      fi
-    fi
+    doDevonCommand python
```

Comment:
I changed the command so that it works for the latest python command. In the latest python command you can't run `devon python pip,` but you can run `pip` individually.

Further in the python command still need to add the script folder to the path.
`PATH=/c/projects/devonfw-ide-scripts-2022.04.003/software/python/Scripts:$PATH

Currently, if you run `devon az` when python is not installed, you have to restart the console again so that pip3 is recognized in the path.
`